### ISSUE TITLE
The first argument to Callable must be a list of types, parameter specification, or "..." when using a `ParamSpec` instance

### ISSUE THREAD

**Issue by multimeric** [python/mypy#13067]
**Bug Report**

`Callable[SomeParamSpec]` throws a spurious error.

(A clear and concise description of what the bug is.)

**To Reproduce**

```python
from typing import Callable, ParamSpec, Any

Spec = ParamSpec("Spec")
T = Callable[Spec, Any]
```

**Expected Behavior**
This should not result in any error.

(Write what you thought would happen.)

**Actual Behavior**

```
main.py:4: error: The first argument to Callable must be a list of types, parameter specification, or "..."
main.py:4: note: See https://mypy.readthedocs.io/en/stable/kinds_of_types.html#callable-types-and-lambdas
```

(Write what happened.)

**Your Environment**

- Mypy version used: Any
- Python version used: Any

Playground: https://mypy-play.net/?mypy=latest&python=3.10&gist=d01c84f502d89076345cd8f8502ec039

**Comment by AlexWaygood** [comment]
Duplicate of #11855
